api/src/main/resources/application.properties

```diff
@@ -1,8 +1,7 @@
 spring.jpa.hibernate.naming.physical-strategy=org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
-# spring.datasource.url = jdbc:mysql://127.0.0.1:3306/BeanBankCoin
-# spring.datasource.url = Add your own: jdbc:mysql://<hostname>:<port>/<database name>
+spring.datasource.url = jdbc:mysql://bean-bank-coin-db-mysql.cbsozziiwdya.eu-west-1.rds.amazonaws.com:3306/bean_bank_coin_db
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 spring.jpa.show-sql: true
 spring.jpa.hibernate.ddl-auto=validate
-# spring.datasource.username=root
-# spring.datasource.password=password
\ No newline at end of file
+spring.datasource.username=admin
+spring.datasource.password=
```

Comment:
Please maybe be more descriptive and use
`spring.datasource.password=<password>`
So that we know that we need to insert that variable